cli/drivers/DrupalValetDriver.php

```diff
@@ -57,7 +57,7 @@ public function frontControllerPath($sitePath, $siteName, $uri)
     {
         $sitePath = $this->addSubdirectory($sitePath);
 
-        if (!isset($_GET['q']) && !empty($uri) && $uri !== '/') {
+        if (!isset($_GET['q']) && !empty($uri) && $uri !== '/' && strpos($uri, '/jsonapi/') === false) {
```

Comment:
👋 I'm the JSON:API maintainer in Drupal.

Bear in mind that the `/jsonapi` path is configurable. So is anyone changes the defaults this may break.

Reply:
@e0ipso oh yeah. I completely missed that. Funny thing is, IIRC, I wrote the patch to make this configurable. 😄

Reply:
Thanks for sharing, y'all! Sooo.. is there any way to fix this driver or is that just not going to be possible since it's totally variable?

Reply:
I have a custom prefix and good to have this issue to know where should I look for in valet. thanks!